src/store/apis/hotels/getPopularHotels.endpoint.js

```diff
@@ -0,0 +1,11 @@
+import { hotelsApi } from './hotels.api';
+
+export const homesApi = hotelsApi.injectEndpoints({
+  endpoints: (builder) => ({
+    getHomesPosts: builder.query({
+      query: () => 'hotels/popular',
+    }),
+  }),
+});
+
+export const { useGetHomesPostsQuery } = homesApi;
```

Comment:
The main suggestion to move api endpoint to existing object and renaming
```suggestion
    getPopularHotels: builder.query({
      query: () => apiEndpoint.popularHotels,
    }),
  }),
});

export const { useGetPopularHotelsQuery } = homesApi;
```